src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -261,3 +261,139 @@
         - name: --remove-action -r
           short-summary: Remove receivers from the action group. Accept space separated list of receiver names.
 """
+
+helps['monitor activity-log alert'] = """
+    type: group
+    short-summary: Manage activity log alerts
+"""
+
+helps['monitor activity-log alert list'] = """
+    type: command
+    short-summary: List activity log alerts under a resource group or the current subscription.
+    parameters:
+        - name: --resource-group -g
+          short-summary: Name of the resource group under which the activity log alerts are being listed. If it is
+                         omitted, all the activity log alerts under the current subscription are listed.
+"""
+
+helps['monitor activity-log alert create'] = """
+    type: command
+    short-summary: Create a default activity log alert
+    long-summary: This command will create a default activity log with one condition which compares if the activities
+                  logs 'category' field equals to 'ServiceHealth'. The newly created activity log alert does not have
+                  any action groups attached to it.
+    parameters:
+        - name: --name -n
+          short-summary: Name of the activity log alerts
+        - name: --scope -s
+          short-summary: A list of resourceIds that will be used as prefixes. The alert will only apply to activityLogs
+                         with resourceIds that fall under one of these prefixes. This list must include at least one
+                         item.
+        - name: --disable
+          short-summary: Disable the activity log alert after it is created.
+        - name: --description
+          short-summary: A description of this activity log alert
+        - name: --condition -c
+          short-summary: Add condition. Accept space separated conditions in 'FIELD=VALUE' format. The possible values
+                         for the field are (case-insensitive) 'resourceId', 'category', 'caller', 'level',
+                         'operationName', 'resourceGroup', 'resourceProvider', 'status', 'subStatus', 'resourceType',
+                         or anything beginning with 'properties.'. If a condition with the same field exists, its value
+                         will be updated.
+        - name: --action -a
+          short-summary: Add action group. Accept space separated resource id of the action groups to be added. The
+                         invalid resource id is ignored.
+        - name: --webhook-properties -w
+          short-summary: Space separated web hook properties in 'key[=value]' format. These properties will be
```

Comment:
This is confusing to me. Examples would help tremendously.  Are these basically the args that are sent to the webhook as query parameters or as part of the body?  Are these only static values?